common/src/test/scala/org/apache/celeborn/common/util/UtilsSuite.scala

```diff
@@ -112,10 +112,6 @@ class UtilsSuite extends CelebornFunSuite {
     assert(1596081225 == Utils.bytesToInt(Array(73.toByte, 68.toByte, 34.toByte, 95.toByte), false))
   }
 
-  test("getThreadDump") {
-    assert(Utils.getThreadDump().nonEmpty)
```

Comment:
ok, this method is covered before as well.

Reply:
But the workload is different, as in ApiWorkResourceSuite, it setup a mini cluster.